frontend/src/app/settings/modelsources/git-settings/edit-git-settings/edit-git-settings.component.ts

```diff
@@ -77,4 +79,8 @@ export class EditGitSettingsComponent implements OnInit, OnDestroy {
         this.router.navigate(['../..'], { relativeTo: this.route })
       );
   }
+
+  navigateBack() {
+    this.location.back();
+  }
```

Comment:
Same as above: I would use `this.router.navigate(['../..'], { relativeTo: this.route })` instead